### ISSUE TITLE
I have no idea how to use this library

### ISSUE THREAD

**Issue by TheLastGimbus** [letsar/flutter_staggered_grid_view#137]
This library seems like a really good way to automatically do some layouts that should be built into Fluttet

### BUT.... I have no idea how to use it. 
The readme.md tells me nothing. There's one actual example (one with green tiles), and I just can't figure out how it works, by experimenting with it, or reading documentation.

The [constructors](https://pub.dev/packages/flutter_staggered_grid_view#constructors) and [tiles](https://pub.dev/packages/flutter_staggered_grid_view#tiles) also doesn't explain me much - I don't get what is a `tile` and `cell`, and how they differ and what is what :confused:

What was my goal is I wanted to have multiple tiles (or, because I don't know what finally a 'tile' is - just rectangle `Widget`s) with undefined height, and they would nicely lay out in one column, or split into multiple columns if the screen was wide enough - I thought that `StaggeredGridView.extent` is what I've been looking for, but it showed some weird behavior (that I couldn't understand because documentation didn't tell me anything), so I finally ended up with `.count` with `crossAxisCount` being calculated by `MediaQuery.of(context).size.width`

### TL;DR what is this issue about?
I think the documentatin/readme just doesn't explain how to really use this and what is what. If someone could write a nice brief description of which element changes what and how to use it - I would be really glad to read it step by step

**Comment by gipic** [comment]
https://romain-rastel.medium.com/flutorial-create-a-staggered-gridview-9c881a9b0b98

**Comment by letsar** [comment]
Is the documentation is better now ? Can you test with the latest 0.5.0 preview? https://pub.dev/packages/flutter_staggered_grid_view/versions/0.5.0-dev.1

**Comment by TheLastGimbus** [comment]
Yeah, readme does look a lot better now :tada: I may not have time to actually play with it, but when briefly reading, it looks like there are lot's of nice examples! Nice job :tada:
